docs/4_the-formal-stuff/2_career-development/6_charts.md

```diff
@@ -1,3 +1,7 @@
+---
+sidebar_position: 6
+---
+
 # Competence Charts
```

Comment:
I think these could be now a part of How to

Reply:
Agree, moved